trigger/testTimer/metadata.go

```diff
@@ -0,0 +1,44 @@
+package testTimer
+
+import "github.com/project-flogo/core/data/coerce"
+
+type Settings struct {
+	Control bool   `md:"control"`
+	Port    string `md:"port"`
+}
+
+type HandlerSettings struct {
+	Id 			   string `md:"id,required"`
+	StartInterval  string `md:"startDelay"`
+	RepeatInterval string `md:"repeatInterval,required"`
+	FilePath       string `md:"filePath"`
+	Header         bool   `md:"header"`
+	Block          bool   `md:"block,required"`
+	Count          int
```

Comment:
You are overloading HandlerSettings here.  It should just be actual settings.  I would suggest making another object which you can use to keep track of the chan, data and count.  Maybe you can have a map for ones with id, and a list of all of them to easily start/stop everything.